views/components/Maps/Maps.tsx

```diff
@@ -13,11 +13,14 @@ import 'leaflet/dist/leaflet.css';
 import "leaflet-defaulticon-compatibility";
 import "leaflet-defaulticon-compatibility/dist/leaflet-defaulticon-compatibility.css";
 import { LatLngExpression, latLng } from "leaflet";
-import { GPSCoordinate } from "@/stores/GPS/GPSTypes";
+import { GPS } from "@/stores/GPS/GPSTypes";
+import { GlobalPath } from "@/stores/GlobalPath/GlobalPathTypes";
 
 export interface IMapsProps {
-    gpsLocation: GPSCoordinate,
-    gpsPath: LatLngExpression[]
+    gpsLocation: GPS,
+    gpsPath: LatLngExpression[],
+    globalPathLocation: GlobalPath,
```

Comment:
Please remove `globalPathLocation`. It's not necessary here and is not being used.

We needed both `gpsLocation` and `gpsPath` to distinguish between the GPS's marker and path. In the case for GlobalPath, we only care about the path